### ISSUE TITLE
Stopping media via Google Assistant (AVRCP)

### ISSUE THREAD

**Issue by Heckie75** [wastis/MediaPlayerRemoteInterface#2]
Hi,

many thanks for providing this addon. I was looking and researching at least two evenings for an addon or service like this, how dbus and MPRIS2 work. I already startet developing a dbus service by my self but failed, since I subscribed to ```AcceleratorActivated``` and after all I saw that ```action-modes``` are not stable and therefore can't be mapped. Now, this is not necessary anymore.

Your addon makes my remote control work perfectly (Zotac remote / Philips (or NXP) Infrared Transceiver, see also my other poor [workaround](https://github.com/Heckie75/kodi-addon-philips-ovu710023-remote) that I had before.

The only thing that is not working is the voice control via my bluetooth speaker with Google Assistant. It is recognized as AVRCP control. When I say "hey Google, stop media" the following message can be seen in ```dbus-monitor```:
```
signal time=1667638216.017319 sender=:1.42 -> destination=(null destination) serial=7145 path=/org/gnome/Shell; interface=org.gnome.Shell; member=AcceleratorActivated
   uint32 184
   array [
      dict entry(
         string "device-name"
         variant             string "Schlafzimmer (AVRCP)"
      )
      dict entry(
         string "action-mode"
         variant             uint32 184
      )
      dict entry(
         string "timestamp"
         variant             uint32 20750872
      )
      dict entry(
         string "device-mode"
         variant             string "/dev/input/event13"
      )
   ]
method call time=1667638216.017797 sender=:1.31 -> destination=:1.107 serial=194 path=/org/mpris/MediaPlayer2; interface=org.mpris.MediaPlayer2.Player; member=Pause
```

Unfortunately this event doesn't seem to be handled by your addon since nothing happen. "Schlafzimmer" is the name of the bluetooth speaker. Do you have an idea what goes wrong?

Other commands seem to work, e.g. when I say "Hey google, next title" Kodi plays next title in playlist and the following can be seen in dbus-monitor:
```
signal time=1667638927.266111 sender=:1.42 -> destination=(null destination) serial=7764 path=/org/gnome/Shell; interface=org.gnome.Shell; member=AcceleratorActivated
   uint32 182
   array [
      dict entry(
         string "timestamp"
         variant             uint32 21462229
      )
      dict entry(
         string "device-mode"
         variant             string "/dev/input/event13"
      )
      dict entry(
         string "device-name"
         variant             string "Schlafzimmer (AVRCP)"
      )
      dict entry(
         string "action-mode"
         variant             uint32 182
      )
   ]
method call time=1667638927.267348 sender=:1.31 -> destination=:1.107 serial=195 path=/org/mpris/MediaPlayer2; interface=org.mpris.MediaPlayer2.Player; member=Next
```

Best regards,
Heckie

BTW: Have you thought providing this addon to the official repos? Or are there any dependencies violating their policies?

**Comment by Heckie75** [comment]
@wastis
After reviewing your code I have seen that this command is mapped to "pause" key. I solved it by using the [KeymapEditor](https://kodi.tv/addons/matrix/script.keymap) and re-mapping the "Google Assistant" command to pause.

my ```.kodi/userdata/keymaps/gen.xml```
```xml
<keymap><global><keyboard><key id="61661">pause</key></keyboard></global></keymap>
```

**Comment by wastis** [comment]
Glad it works for you.  Also happy to see it works with the google assistant of your speaker, I did not test it as I do not have such a device.

Yes, I thought about a pull request for this. To my knowledge, the code complies 100% to the policies, I spend quite a time to make all my addons compliant. There is some not negligible effort required to create a proper pull request and I was discouraged creating any further pull requests with kodi addon repository when I made the pull request for the equalizer addon not receiving any comment by the maintainers for 7 months, they simply ignored my pull request. So for me it is not worthwhile.
